bundle.yaml.j2

```diff
@@ -88,7 +90,17 @@ relations:
 {% endif -%}
 # COS-monitoring
 - - traefik:metrics-endpoint
-  - prometheus
+  - prometheus:metrics-endpoint
+- - alertmanager:self-metrics-endpoint
+  - prometheus:metrics-endpoint
+- - loki:metrics-endpoint
+  - prometheus:metrics-endpoint
+- - loki:grafana-dashboard
+  - grafana:grafana-dashboard
+- - prometheus:grafana-dashboard
+  - grafana:grafana-dashboard
+- - alertmanager:grafana-dashboard
+  - grafana:grafana-dashboard
```

Comment:
Since this section is very structured, I would rewrite it a bit to increase clarity:
```yaml
- [prometheus:metrics-endpoint, traefik:metrics-endpoint]
- [prometheus:metrics-endpoint, alertmanager:self-metrics-endpoint]
- [prometheus:metrics-endpoint, loki:metrics-endpoint]
- [grafana:grafana-dashboard, loki:grafana-dashboard]
- [grafana:grafana-dashboard, prometheus:grafana-dashboard]
- [grafana:grafana-dashboard, alertmanager:grafana-dashboard]
```